configs/balrogworker

```diff
@@ -0,0 +1,25 @@
+{
+    "us-east-1": {
+        "type": "balrog-worker",
+        "domain": "srv.releng.use1.mozilla.com",
+        "ami": "ami-18abe670",
+        "subnet_ids": ["subnet-33a98358"],
+        "security_group_ids": ["sg-cf1027ab"],
+        "instance_type": "t2.micro",
+        "distro": "centos",
+        "ssh_key": "aws-releng",
+        "use_public_ip": true,
+        "instance_profile_name": "balrog-worker",
```

Comment:
`instance_profile_name` is a IAM role (a set of permissions, for this instance). Do you need this? If not, please check if the `aws_create_instace.py` script handle the case, when `instance_profile_name` is missing.